README.md

```diff
@@ -57,6 +57,7 @@ The setup role will do the following,
   - By default, Receptor is obtained via the pre-built binary on the Receptor Github release page.
   - For Centos/Redhat/Fedora systems, Receptor can be installed via the `dnf` package manager.
   - It is also possible to upload a custom Receptor binary from the local filesystem to the node.
+  - It is also possible to install receptor via [container image](https://quay.io/repository/ansible/receptor?tab=tags&tag=latest).
```

Comment:
Shoud it be `https://quay.io/repository/ansible/awx-ee?tab=tags&tag=latest`?

Seems `quay.io/ansible/awx-ee` is used for `receptor_image_name` by default. `quay.io/ansible/receptor` can't be used since it does not contain ansible-runner.

Reply:
Just fixed!